docs/nx/next/infra.md

```diff
@@ -0,0 +1,173 @@
+### @ensono-stacks/next:infrastructure
+
+<details>
+<summary>Configure Infrastructure for your Next project</summary>
+
+The infrastructure generator will provide all the necessary tools and setup ready to configure your pipeline.
```

Comment:
Technically not configuring a pipeline at all here. More infrastructure required to host your application in a Kubernetes Cluster.